README.md

```diff
@@ -101,6 +110,25 @@ Set these environment variables when debugging:
 RUST_LOG=lorri=debug RUST_BACKTRACE=1 lorri watch
 ```
 
+### `lorri` reevaluates more than expected
+
+`lorri` sometimes recursively watches a directory that the user did
+not expect. This can happen for a number of reasons:
+
+1. when using a directory instead of a channel for `nixpkgs`, `lorri`
+   watches that directory recursively, including the potential `.git`
+   directory and its subdirectories
+2. when specifying `src` via a path, `lorri` watches that path
+   recursively (see https://github.com/target/lorri/issues/6
+   for details). To get around this, use a `filterSource`-based
+   expression to specify `src`, e.g., use
+   [nix-gitignore](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/nix-gitignore/default.nix):
+   `src = pkgs.nix-gitignore.gitignoreSource [] ./.` (note that the
+   trailing `./.` will not be watched by `lorri`)
+3. when using a construct like `import ./.` to import a `default.nix`
+   file, `lorri` watches the current directory recursively. To get
+   around it, use `import ./default.nix`.
```

Comment:
Is that true? Normally `import ./.` is an alias for `./default.nix`, so I’m suprised this happens.

Reply:
Interesting, I'll need to explore this.

Reply:
I forgot to post my findings, but _yes_ this is true! Nix doesn't emit `/path/to/default.nix` as the true file, it emits just '/path/to`. A possible solution here is to append  a `/default.nix` when an imported directory is identified.

Reply:
Okay, opened a separate issue.

Reply:
I believe this is now fixed: https://github.com/target/lorri/blob/master/src/builder.rs#L160-L170

See https://github.com/target/lorri/pull/249